lib/YaST/NetworkSettings/NetworkCardSetup/BondSlavesTab.pm

```diff
@@ -35,6 +37,7 @@ sub select_tab {
 
 sub select_bond_slave_in_list {
     assert_screen(BOND_SLAVES_TAB);
+    apply_workaround_bsc1204176(BOND_SLAVE_DEVICE_CHECKBOX_UNCHECKED) if (is_sle('>=15-SP4'));
```

Comment:
(optional) you don't need the surrounding parenthesis, just:
`is_sle('>=15-SP4')`